app/javascript/packs/commentModPage.js

```diff
@@ -1,5 +1,8 @@
+import { reactionsPath } from '../routes.js.erb';
```

Comment:
When I try exactly same solution for `app/javascript/packs/toggleUserFlagModal.js` the tests fail with following error:
```
    Details:

   /app/javascript/routes.js.erb:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){<%= JsRoutes.generate %>
                                                                                      ^

    SyntaxError: Unexpected token '<'

    > 1 | import { reactionsPath } from '../routes.js.erb';

```